components/webgpu/lib.rs

```diff
@@ -82,12 +83,14 @@ pub enum WebGPURequest {
     },
     CreateBindGroup {
         device_id: id::DeviceId,
+        scope_id: Option<u64>,
```

Comment:
let's add a TODO to consider using `NonZeroU64` to reduce size of the enum by 8 bytes